pkg/codegen/templates/server-interface.tmpl

```diff
@@ -1,6 +1,6 @@
 // ServerInterface represents all server handlers.
 type ServerInterface interface {
```

Comment:
How can this ever produce an interface named something other than ServerInterface?